src/parquet/util/bit-util.h

```diff
@@ -33,6 +33,14 @@ namespace parquet_cpp {
 // TODO: is this in boost or something else like that?
```

Comment:
This comment is straight from the Impala codebase. We should review the bit utilities at some point and see if we can use 3rdparty code (if it's better-tested / more portable) to replace some of these